src/mcp_agent/config.py

```
@@ -219,26 +223,65 @@ class UsageTelemetrySettings(BaseModel):
-    """Log spans to console"""
-
     sample_rate: float = 1.0
     """Sample rate for tracing (1.0 = sample everything)"""
 
+    otlp_settings: TraceOTLPSettings | None = None
+    """OTLP settings for OpenTelemetry tracing. Required if using otlp exporter."""
+
+    # Settings for advanced trace path configuration for file exporter
+    path_settings: TracePathSettings | None = None
+    """
+    Save trace files with more advanced path semantics, like having timestamps or session id in the trace name.
```

Pylint (3.3.7)

[convention] Line too long (111/100)

(C0301)